contracts/access/AccessControlDefaultAdminRules.sol

```diff
@@ -222,19 +282,37 @@ abstract contract AccessControlDefaultAdminRules is IAccessControlDefaultAdminRu
         super._revokeRole(role, account);
     }
 
+    /**
+     * @dev Sets a pending delay into effect if its delay has passed
+     */
+    function _materializeDefaultAdminDelayChange() private {
+        uint48 delaySchedule = _pendingDelaySchedule;
+        if (_isSet(delaySchedule) && _hasPassed(delaySchedule)) _currentDelay = _pendingDelay;
+    }
+
     /**
      * @dev Resets the pending default admin and delayed until.
      */
-    function _resetDefaultAdminTransfer() private {
-        delete _pendingDefaultAdmin;
-        delete _defaultAdminTransferDelayedUntil;
+    function _rollbackDefaultAdminDelay() private {
+        _materializeDefaultAdminDelayChange();
+
+        delete _pendingDelay;
+        delete _pendingDelaySchedule;
     }
 
     /**
-     * @dev Checks if a {defaultAdminTransferDelayedUntil} has been set and passed.
+     * @dev Resets the pending default admin and delayed until.
      */
-    function _hasDefaultAdminTransferDelayPassed() private view returns (bool) {
-        uint48 delayedUntil = defaultAdminTransferDelayedUntil();
-        return delayedUntil > 0 && delayedUntil < block.timestamp;
+    function _resetDefaultAdminTransfer() private {
+        delete _pendingDefaultAdmin;
+        delete _pendingDefaultAdminSchedule;
+    }
+
+    function _hasPassed(uint48 schedule) private view returns (bool) {
+        return schedule < block.timestamp;
+    }
+
+    function _isSet(uint48 schedule) private pure returns (bool) {
```

Comment:
I would rename this to `_isScheduleSet`, and `_hasPassed` to `_hasSchedulePassed`.

Reply:
Done :)